rllib/utils/actor_manager.py

```diff
@@ -199,84 +248,85 @@ def num_actors(self) -> int:
     @DeveloperAPI
     def num_healthy_actors(self) -> int:
         """Return the number of healthy remote actors."""
-        return sum(self.__remote_actor_states)
+        return sum(
+            [1 for s in self.__remote_actor_states.values() if s.is_healthy]
+        )
 
     @DeveloperAPI
     def num_outstanding_async_reqs(self) -> int:
         """Return the number of outstanding async requests."""
-        return len(self.__in_flight_req_to_actor_idx)
+        return len(self.__in_flight_req_to_actor_id)
 
     @DeveloperAPI
-    def is_actor_healthy(self, idx: int) -> bool:
+    def is_actor_healthy(self, actor_id: int) -> bool:
         """Whether a remote actor is in healthy state.
 
         Args:
-            idx: Index of the remote actor.
+            actor_id: ID of the remote actor.
 
         Returns:
             True if the actor is healthy, False otherwise.
         """
-        return self.__remote_actor_states[idx]
+        return self.__remote_actor_states[actor_id].is_healthy
```

Comment:
what if the actor_id does not exist? this should give a descriptive error?

Reply:
sure. done.